### PR TITLE
Update WebKit to version 12.3

### PR DESCRIPTION
**Pull request by mbektchiev**

### CODE REVIEW

**Comment by Natalia-Hristova** [comment]
test --performance-only

**Comment by Natalia-Hristova** [comment]
test --performance-only

**Comment by Natalia-Hristova** [comment]
test --performance

**Comment by mbektchiev** [comment]
test

File: src/WebKit.cmake [hunk 1]
```diff
@@ -11,11 +11,13 @@ set(WEBKIT_CMAKE_ARGS
     -DCMAKE_SYSTEM_PROCESSOR=arm
     -DCMAKE_XCODE_ATTRIBUTE_SUPPORTED_PLATFORMS=${CMAKE_XCODE_ATTRIBUTE_SUPPORTED_PLATFORMS}
     -DCMAKE_XCODE_EFFECTIVE_PLATFORMS=${CMAKE_XCODE_EFFECTIVE_PLATFORMS}
+    -DCMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET=${CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET}
```
Comment (vtrifonov) [inline src/WebKit.cmake]: Why are we adding these additional flags?

Comment (mbektchiev) [reply src/WebKit.cmake]: `CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET` is necessary in order to force JSC to match the deployment target of NativeScript. Without it, it defaulted to latest (e.g. 12.4 with Xcode 10.2.1) and that resulted in compiler or maybe linker errors.

**Pull request merged**